DeploymentCloud/Deployment.DataX/Samples/samples/iotDevice/iotsample.json

```diff
@@ -4,7 +4,7 @@
 		{
 			"dataTypeName": "DeviceData",
 			"simulationPeriodInMinute": 1,
-			"numEventsPerBatch": 100,
+			"numEventsPerBatch": 50,
```

Comment:
how many events is that per month, will that work well with the change above?  Can we document these changes either in this PR or in our tutorials/wiki pages?

Reply:
With this change, about 78,000 events will be sent. It will be good enough. Worked with Rohit for this.